nvi

```diff
@@ -165,6 +165,14 @@ install_node() {
       -- \
       "$INSTALL_DIR/$PACKAGE/bin/npm" \
       "$EXEC_DIR/npm"
+    # npx has only been included since npm 5.2.0 so we avoid copying it if it does not exist.
```

Comment:
This is in the realm of nit-picks, but technically we never attempt to copy anything. We skip _linking_ it if it doesn't exist. You could argue it's been copied either way with the whole downloading from the internet and extracting to disk, so I think we should change the terminology.

Reply:
That is absolutely true, I wrote copying and changed it to "linking it" a lot of places, but obviously not here, where I didn't remove it before pushing. 

![NotLikeThis](https://static-cdn.jtvnw.net/emoticons/v1/58765/3.0)